test/projects/ProfilingTest/Platformer/src/Platformer.jl

```diff
@@ -3,23 +3,32 @@ module Platformer
     using JulGame.Math
     using JulGame.SceneBuilderModule
     
-    JulGame.MAIN.testMode = true
-    JulGame.MAIN.testLength = 30.0
     function run()
+        JulGame.MAIN.testMode = true
+        JulGame.MAIN.testLength = 30.0
+        JulGame.MAIN.currentTestTime = 0.0
```

Comment:
The fact that I need to do this is an issue. It's because of the way we instantiate main. It is done once, so it is shared between projects that are opened in the same session. I think this is why we can't run the same project back to back. Ideally we shouldn't need to change the value of `currentTestTime` here. It should just be 0.